src/library.js

```diff
@@ -2700,6 +2713,14 @@ LibraryManager.library = {
   emscripten_longjmp: function(env, value) {
     _longjmp(env, value);
   },
+#if WASM_BACKEND == 1
+  // We need this wrapper from llvm wasm backend, but this is generated only
+  // in asm.js compilation, so.
```

Comment:
Yes, in the asm.js path the `__wrapper` functions are created so that a JS library function can be called indirectly. In other words, we import `emscripten_longjmp`, and we can't put that in the function table (because asm.js doesn't like that, it wants to know the type), so we wrap it in an asm.js function.